api/src/main/java/org/openmrs/module/eptsreports/ColumnParameters.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.eptsreports;
+
+/**
+ * Show the parameters that can be passed to a column for disaggregation
+ */
+public class ColumnParameters {
```

Comment:
Seems to be specific to datasets, correct?

Can we maybe look at adding this logic to a `BaseDataSet.java` file so that the logic is shared for all datasets? Could be an inner class or a global array with a function for adding columns to a `dataSetColumns` global array.

Reply:
Agreed. I will creat a nested class which is a nicer way of logically grouping classes that are only used in one place, increases encapsulation and It can lead to more readable and maintainable code. Thanks for this point @nthfloor